datumaro/components/extractor.py

```diff
@@ -360,6 +367,38 @@ def as_polygon(self):
     def get_area(self):
         return 0
 
+
+@attrs
+class Cuboid(Annotation):
+    _type = AnnotationType.cuboid
+    points = attrib(type=list, default=[])
+    label = attrib(converter=attr.converters.optional(int),
+                   default=None, kw_only=True)
+    z_order = attrib(default=0, validator=default_if_none(int), kw_only=True)
+
+    def as_cuboid(self):
+        return self.points[:]
+
+    def get_area(self):
+        return 0
+
+
+@attrs
+class Tag(Annotation):
+    _type = AnnotationType.tag
+    label_id = attrib(type=int, default=None)
+    name = name = attrib(default=None, kw_only=True)
+
+
+@attrs
+class Owner(Annotation):
```

Comment:
I think, it is better to use annotation attributes for this kind of information.

Reply:
Resolved